share/completions/minikube.fish

```diff
@@ -0,0 +1,79 @@
+function __minikube_no_command
+    set -l cmd (commandline -poc)
+    set -e cmd[1]
+    if test (count $cmd) -eq 0
+        return 0
+    end
+    return 1
+end
+
+function __minikube_using_command
+    set cmd (commandline -poc)
+    if test (count $cmd) -gt 1
+        if test $argv[1] = $cmd[2]
+            return 0
+        end
+    end
+    return 1
+end
+
+complete -c minikube -f -n "__minikube_no_command" -a addons -d "Modify minikube's kubernetes addons"
+complete -c minikube -f -n "__minikube_no_command" -a completion -d "Outputs minikube shell completion for the given shell (bash)"
+complete -c minikube -f -n "__minikube_no_command" -a config -d "Modify minikube config"
+complete -c minikube -f -n "__minikube_no_command" -a dashboard -d "Opens/displays the kubernetes dashboard URL for your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a delete -d "Deletes a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a docker-env -d "sets up docker env variables; similar to '\$(docker-machine env)'"
+complete -c minikube -f -n "__minikube_no_command" -a get-k8s-versions -d "Gets the list of available kubernetes versions available for minikube."
+complete -c minikube -f -n "__minikube_no_command" -a ip -d "Retrieve the IP address of the running cluster."
+complete -c minikube -f -n "__minikube_no_command" -a logs -d "Gets the logs of the running localkube instance, used for debugging minikube, not user code."
+complete -c minikube -f -n "__minikube_no_command" -a service -d "Gets the kubernetes URL(s) for the specified service in your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a ssh -d "Log into or run a command on a machine with SSH; similar to 'docker-machine ssh'"
+complete -c minikube -f -n "__minikube_no_command" -a start -d "Starts a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a status -d "Gets the status of a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a stop -d "Stops a running local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a version -d "Print the version of minikube."
+complete -c minikube -l "alsologtostderr" -d "log to standard error as well as files"
+complete -c minikube -l "log_backtrace_at" -d "when logging hits line file:N, emit a stack trace (default :0)"
+complete -c minikube -l "log_dir" -d "If non-empty, write log files in this directory (default \"\")"
```

Comment:
What do you suggest we write here, perhaps unnecessary to state the behavior of empty value. Just say something like `Log files in this directory`.